packages/builder/src/components/portal/overview/automation/HistoryTab.svelte

```diff
@@ -145,6 +159,18 @@
           options={statusOptions}
         />
       </div>
+      {#if isEnabled(TENANT_FEATURE_FLAGS.LICENSING) && licensePlan?.type === "free"}
+        <div class="pro-upgrade">
+          <div class="pro-copy">Store up to 30 days of automations</div>
```

Comment:
This message could probably be updated, we have more options now than just 30 days: 

![Screenshot 2022-09-28 at 13 17 20](https://user-images.githubusercontent.com/8755148/192776660-df37c2f4-9cb1-4aa2-8484-27e9e59b69ec.png)

Probably the same for the filters, they can go up to 90 days in cloud or unlimited on enterprise.